include/circt/Dialect/FIRRTL/FIRRTLStructure.td

```diff
@@ -100,7 +100,7 @@ def FModuleOp : FIRRTLOp<"module", [IsolatedFromAbove, Symbol, SingleBlock,
 
     /// Erases the ports listed in `portIndices`.  `portIndices` is expected to
```

Comment:
Sounds good, I took your suggestion here, and I updated one other comment in https://github.com/llvm/circt/pull/3837/commits/24ef0643232a2ebf29a7163cc094cb831cf3a16d.

I left `InstanceOp::erasePorts()`'s comment alone, since I think it was generic enough to still be applicable to BitVectors, but let me know if you think I should make it explicit that set bits in the BitVector correspond to the indices of ports to remove. All other methods didn't have BitVectors in their function signatures and internally constructed the lists of indices to drop, so I didn't need to update comments for most of the code I touched.